### ISSUE TITLE
Integration of rflint to this project?

### ISSUE THREAD

**Issue by aaltat** [andriyko/sublime-robot-framework-assistant#24]
Ugh

I have been looking the rflint [1] for some time and it is starting to look quite good. After I did read the sublime text plugin guide, it should be relatively easy to write new plugin to sublime which supports some basic rflint commands. 

But instead of creating yet another plugin, I think it would be really cool if I could integrate the work to this plugin. So would it would be possible for you to accept pull request that would integrate the rflint to this plugin? 

Example something like this:
```
import sublime, sublime_plugin
from subprocess import Popen
from subprocess import PIPE

class RunRfLintToOpenTabCommand(sublime_plugin.TextCommand):
    def run(self, edit):
        file_name = self.view.file_name()
        result = self._run_fr_lint(file_name)
        curr_view = self.view.window().new_file()
        curr_view.insert(edit, 0, result)

    def _run_fr_lint(self, file_name):
        proc = Popen(['python', '-m', 'rflint', file_name], shell=True, stdout=PIPE)
        return proc.stdout.read().decode("utf-8")
```

[1] https://github.com/boakley/robotframework-lint

**Comment by andriyko** [comment]
Done:
https://github.com/andriyko/sublime-robot-framework-assistant/pull/25
Sublime has so called "Build System" which allows to run external commands and get their output.
```
which rflint
rflint is /Users/andriyko/.virtualenvs/robotframework/bin/rflint
```
[1]. `Robot Framework > Edit rflint Config`
```
{
    "cmd": ["/Users/andriyko/.virtualenvs/robotframework/bin/rflint", "$file"],
    "selector": "text.robot",
}
```
[2]. `Tools > Build System > rflint`
[3]. `Tools > Build`
![screenshot 2015-02-22 21 14 26](https://cloud.githubusercontent.com/assets/719450/6319954/b8940920-bad9-11e4-8b8b-02f7b39b02f0.png)

@aaltat , please let me know if that works for you.

**Comment by aaltat** [comment]
Cool, I did not know that this example could have been achieved in this simple way but I did actually did have more complex examples in my mind. 

I do not want always run the same set of rules for all the test data. When creating high level keywords to be used in test cases, I want run different set of rules and/or configure the rules in different ways. Example keyword documentation may not be required on high level keywords, but lower level keywords it is definitely needed. And I might want to configure the keyword max length to be different, depending where in the test data it is located. Also I was hoping to do some sort of git integration, example run rflint for all changed/staged/not tracked files on my work space. So changing to the build configuration for all the above requirements is not so easy and may not possible.

But in sort term, your solution is elegant and provides me easy way to access the rflint from sublime. 

Note to my self: Remember to explain the long term goal, _always_, when providing simple example what I want in the short term.
